File: app/src/main/java/com/ddiehl/android/reversi/view/LauncherActivity.java
package com.ddiehl.android.reversi.view;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;

import timber.log.Timber;

public class LauncherActivity extends BaseActivity {

    @Override
    protected Fragment createFragment() {
        return new LauncherFragment();
    }

    @Override
    public void onSignInFailed() {
        Timber.d("Sign In FAILED: %s", getGameHelper().getSignInError().toString());
    }

    @Override
    public void onSignInSucceeded() {
        Timber.d("Sign In SUCCESS");
    }

    @Override
    public void onConnected(@Nullable Bundle bundle) {
        Timber.d("onConnected");
    }

    @Override
    public void onConnectionSuspended(int i) {
        Timber.d("onConnectionSuspended");
    }
}

File: app/src/main/java/com/ddiehl/android/reversi/view/LauncherActivity.kt
package com.ddiehl.android.reversi.view

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v7.app.AppCompatActivity

import com.ddiehl.android.reversi.R

class LauncherActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fragment)

        val fm = supportFragmentManager
        var fragment: Fragment? = fm.findFragmentById(R.id.fragment_container)

        if (fragment == null) {
            fragment = LauncherFragment()
            fm.beginTransaction()
                    .add(R.id.fragment_container, fragment)
                    .commit()
        }
    }
}
